.chloggen/3242-rollback-of-3187.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. collector, target allocator, auto-instrumentation, opamp, github action)
+component: target allocator
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Rollback 3187
```

Comment:
We should at least do `#3187`, so this renders into a link to the PR in the changelog.

Reply:
Oh, I wasn't sure about that since I thought it might consider it a line comment. Looks like it already got merged.